docs/v2-roadmap.rst

```diff
@@ -80,15 +80,15 @@ verification in Python or OpenSSL will be immediately
 available.
 
 
-**✨ Optimized for Python 3.6+**
+**✨ Optimized for Python 3.7+**
 --------------------------------
 
 In v2.0 we'll be specifically be targeting
-CPython 3.6+ and PyPy 7.0+ (compatible with CPython 3.6)
-and dropping support Python versions 2.7 and 3.5.
+CPython 3.7+ and PyPy 7.3.2+ (compatible with CPython 3.7)
+and dropping support Python versions 2.7 and 3.6.
```

Comment:
```suggestion
CPython 3.7+ and PyPy 7.3.10+ (compatible with CPython 3.8)
and dropping support Python versions 2.7 and 3.6.
```

Since PyPy stopped supporting CPython 3.7, I think it's OK for us to do the same, even if we do test PyPy with CPython 3.7 support right now.